R/opts.R

```diff
@@ -471,18 +480,22 @@ obs_opts <- function(family = "negbin",
                      week_effect = TRUE,
                      week_length = 7,
                      scale = list(),
+                     na = "missing",
```

Comment:
```suggestion
                     na = c("missing", "accumulate"),
```
Any reason not to handle it like this?

Reply:
Oh I thought we were doing it the other way round but looking at #512 we aren't. I agree that this is clearer.